owner/manager.go

```diff
@@ -120,6 +122,8 @@ func setManagerSessionTTL() error {
 func NewSession(ctx goctx.Context, logPrefix string, etcdCli *clientv3.Client, retryCnt, ttl int) (*concurrency.Session, error) {
 	var err error
 	var etcdSession *concurrency.Session
+	failedCnt := 0
+	intervalCnt := int(3 * time.Second / newSessionRetryInterval)
```

Comment:
Why not define `intervalCnt` as constant as well?
And I think `logInterval` maybe a better name.